<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstLogin</class>
 <widget class="QWidget" name="FirstLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FirstLogin</string>
  </property>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>158</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background:transparent;border-width:0;border-color:white;border-bottom-width:1;border-style:outset;
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>257</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background:transparent;border-width:0;border-color:white;border-bottom-width:1;border-style:outset;
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);font: 16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>设置您的登录口令：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLabel2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>219</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);font: 16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>再输一次：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>318</y>
     <width>260</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
border-radius:25px;
background-color: rgb(255, 255, 255);
color: rgb(119,119,119);
font: 14px &quot;微软雅黑&quot;;
}
QPushButton::hover{background-color: rgb(140,235,231);}

</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>377</y>
     <width>228</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel2">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>153</y>
     <width>267</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="correctLabel">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>167</y>
     <width>16</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="correctLabel2">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>266</y>
     <width>16</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
